app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AdminsController < ApplicationController
```

Comment:
I believe this should be `AdminController` without the 's' — see also my comments on routes